crates/holochain/src/conductor/config.rs

```diff
@@ -157,32 +157,39 @@ pub mod tests {
 
     #[test]
     fn test_config_complete_config() {
-        let toml = r#"
-    environment_path = "/path/to/env"
-    use_dangerous_test_keystore = true
-
-    [passphrase_service]
-    type = "cmd"
-
-    encryption_service_uri = "ws://localhost:9001"
-    decryption_service_uri = "ws://localhost:9002"
-    signing_service_uri = "ws://localhost:9003"
-
-    [dpki]
-    instance_id = "some_id"
-    init_params = "some_params"
-
-    [[admin_interfaces]]
-    driver.type = "websocket"
-    driver.port = 1234
-
-    [[network.transport_pool]]
-    type = 'proxy'
-    sub_transport = { type = 'quic', bind_to = 'kitsune-quic://0.0.0.0:0' }
-    proxy_config = { type = 'local_proxy_server', proxy_accept_config = 'reject_all' }
+        let yaml = r#"---
+    environment_path: /path/to/env
+    use_dangerous_test_keystore: true
+    signing_service_uri: ws://localhost:9001
+    encryption_service_uri: ws://localhost:9002
+    decryption_service_uri: ws://localhost:9003
+
+    passphrase_service:
+      type: cmd
+
+    dpki:
+      instance_id: some_id
+      init_params: some_params
+
+    admin_interfaces:
+      -
+        driver:
+          type: websocket
+          port: 1234
+
```

Comment:
Is this a array of length one where the value is a dictionary of two key pairs?
Shouldn't it be:
```suggestion
      - driver:
          type: websocket
          port: 1234

```
Or does it make no difference?

Reply:
Yeah, I think either way is fine, but using one less line may be more canonical.